### ISSUE TITLE
Error in SAIGE-GENE - step2

### ISSUE THREAD

**Issue by andronekomimi** [weizhouUMICH/SAIGE#263]
Hello,
I got the following error message in **SAIGE-GENE** Step2_SPAtests:

```
Error in LU.dgC(a) : cs_lu(A) failed: near-singular A (or out of memory)
Calls: SPAGMMATtest ... solve -> .local -> solve.dsC.mat -> .solve.dgC.lu -> LU.dgC
Timing stopped at: 0.084 0.008 0.11
Execution halted
```
Here is my command-line for a quantitative trait:

```
Rscript step2_SPAtests.R \
--vcfFile=extract.vcf.gz \
--vcfFileIndex=extract.vcf.gz.tbi \
--vcfField=GT \
--chrom=1 \
--minMAF=0 \
--GMMATmodelFile=saige-gene_step1.rda \
--varianceRatioFile=saige-gene_step1_cate.varianceRatio.txt \
--SAIGEOutputFile=saige-gene_step2.txt \
--numLinesOutput=1 \
--groupFile=extract.grp.txt \
--sparseSigmaFile=sparseGRM_relatednessCutoff_0.125_2000_randomMarkersUsed.sparseGRM.mtx \
--IsSingleVarinGroupTest=TRUE \
--IsDropMissingDosages=TRUE
```

All missing phenotypes have been removed.

R version 4.0.2
SAIGE version SAIGE_0.39

Thank you

**Comment by sariya** [comment]
@andronekomimi  Hi - Were you able to resolve this?

**Comment by weizhouUMICH** [comment]
Sorry fo the late reply! We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
